ArduPlane/quadplane.cpp

```diff
@@ -120,14 +120,14 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
 
     // YAW_RATE_MAX index 25
 
-    // @Param: LAND_SPEED
-    // @DisplayName: Land speed
-    // @Description: The descent speed for the final stage of landing in cm/s
-    // @Units: cm/s
-    // @Range: 30 200
-    // @Increment: 10
+    // @Param: LAND_FINAL_SPD
```

Comment:
I'd vote for LAND_SPEED_FINAL because "FINAL" is really more like "MAX", "MIN" and "LIMIT"

Reply:
its to match the existing LAND_FINAL_ALT

Reply:
ok, well, if you think of LAND_FINAL as a sub feature (or stage) of landing then it makes sense.